src/reducers/LoginReducer.js

```diff
@@ -3,6 +3,7 @@ import { LOGIN_START, LOGIN_SUCCESS, LOGIN_ERROR } from "../actions/Login";
 export const initialState = {
   email: "",
   token: "",
+  message:"",
```

Comment:
we should take this out, no? same with line 22

Reply:
I will remove references to message thanks!